app/controllers/client_data_controller.rb

```diff
@@ -94,12 +94,20 @@ def auth_errors(exception)
     path = polymorphic_path(model_class, action: :new)
     # prevent redirect loop
     if path == request.path
-      render "authorization_error", status: 403, locals: { msg: exception.message }
+      render_auth_errors(exception)
     else
       redirect_to path, alert: exception.message
     end
   end
 
+  def render_auth_errors(exception)
+    if exception.message == "Client is disabled"
+      render_disabled_client_message(exception.message)
```

Comment:
Because there are several sets of `if/else` if was a bit tricky to follow, here is my thinking in English:

ok, so IF the exception message is "Client is disabled" then `render_client_message` ... goes to that method and it calls `  render "#{current_user.client_slug}/_disabled", status: 403` but then rescues that to return `render "authorization_error", status: 403, locals: { msg: exception.message }` which is the SAME as the `else` here. so if we just called `render_disabled_client_message(exception.message)` here, wouldn't that yield the same result?